RDKShell/CHANGELOG.md

```diff
@@ -15,6 +15,10 @@ All notable changes to this RDK Service will be documented in this file.
 * Changes in CHANGELOG should be updated when commits are added to the main or release branches. There should be one CHANGELOG entry per JIRA Ticket. This is not enforced on sprint branches since there could be multiple changes for the same JIRA ticket during development. 
 
 * For more details, refer to [versioning](https://github.com/rdkcentral/rdkservices#versioning) section under Main README.
+## [1.5.0] - 2023-09-22
+### Fixed
+- Added Factory-hal api
```

Comment:
We have not added Factory-hal API.
Replace it as: Updating getEthernetMac usage as per factory-hal code review recommendation.